runtime/lua/vim/treesitter/_query_linter.lua

```diff
@@ -1,20 +1,20 @@
 local api = vim.api
 
 local namespace = api.nvim_create_namespace('vim.treesitter.query_linter')
--- those node names exist for every language
-local BUILT_IN_NODE_NAMES = { '_', 'ERROR' }
 
 local M = {}
 
 --- @class QueryLinterNormalizedOpts
 --- @field langs string[]
 --- @field clear boolean
 
+--- @alias ParseError {msg: string, range: Range4}
```

Comment:
Make sure to namespace types appropriately. Use `vim.` at minimum.

Reply:
It's a shame LuaLs doesn't have real namespacing yet. The verbose type names can be a little annoying for simple aliases.